velox/expression/RowConstructor.cpp

```diff
@@ -32,35 +32,26 @@ TypePtr RowConstructorCallToSpecialForm::resolveType(
 }
 
 ExprPtr RowConstructorCallToSpecialForm::constructSpecialForm(
-    const std::string& name,
```

Comment:
Hi @rui-mo, thank you for making this change! I wonder if it would be better to let this function still receive a name as argument, in case there will be other engines (besides Presto and Spark) in the future that want to register this function with a different name?